src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -31,7 +31,7 @@
 
 helps['consumption reservations details list'] = """
     type: command
-    short-summary: List reservations details by reservation order id and optionally by reservation id
+    short-summary: List the details of a reservation by reservation order id and optionally by reservation id.
```

Comment:
Avoid the word `optionally`, instead use `or` in help strings:

`List the details of a reservation by order ID or reservation ID.`

Reply:
Resolved